config/whatsnew.json

```diff
@@ -0,0 +1,9 @@
+{
+  "13.0.1" : {
+    "en" : [
+      "Pollute the files failing to update l n files for pdf",
+      "Emaillogin server show open graph preview in gallery removed old code",
+      "Server sharee email search server log exceptions that s settings server"
```

Comment:
Why not ship markdown directly? Then we could also format it a bit better.

Reply:
But has the downside of a potential code injection.

Reply:
It's safer,  and simpler. Imho, thee short lines can live well without formatting. @jancborchardt ?

Reply:
We want to display it really simply, boiled down to the essence. Like in the updates screen of Android Play Store or iOS Appstore. There’s no formatting.